### ISSUE TITLE
Chat text input field accessibility, with key combination is unstable

### ISSUE THREAD

**Issue by chasapakis** [element-hq/element-web#28086]
### Steps to reproduce

The keyboard shortcut e or Alt+e jumps to input fields in HTML documents and applications such as Element. In Element, however, this only works unreliably for the chat input field, I can jump to the field, but then I can't get into the field with Enter or Space to enter text. I can only get into the emoji search field, which probably comes directly after the chat field. If I jump to the next unread message with Shift+Alt+Arrow down, I land directly in the input field of the corresponding chat, but as soon as I leave the field with Escape to read the messages, I cannot get back into the field to reply.

Sometimes, after jumping back and forth several times with e and Alt+e or with the arrow keys, I do manage to enter the field, but seems like a matter of "luck" in the sense that I could not manage to find the pattern behind this behaviour. The only reliable way to get into the chat input field is to search for the desired person or group with **Ctrl+k**. This then takes me straight back into the field and I can write a reply.

### Outcome

I try to Reach the text input field to send message using key combination reliably.
But I cannot reach the text input field to reply in a stable way.

Sometimes it works smoothly for a while and I reach the field, but sometimes I get stuck every time I try to reply. I have not yet been able to find out what this depends on. 

### Operating system

Windows

### Application version

1.11.77

### How did you install the app?

From element.io

### Homeserver

Synapse 1.99.0

### Will you send logs?

No

**Comment by chasapakis** [comment]
Hi all, sorry for the duplicate. The original question was regarding the desktop-app though. I don´t know if this affects element-web too. When I submitted the question in desktop-app it ended up originally here, or I just messed up something. I just hope here is there right place and not rather the desktop-app issue tracker as this was observed in the desktop-app.
